.tekton/execute-doc-ingestion-task.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: execute-doc-ingestion-pipeline
+spec:
+  workspaces:
+    - name: output
+  params:
+    - name: WORK_DIRECTORY
+      description: Directory to start build in (handle multiple branches)
+      type: string
+    - name: APPLICATION_NAME
+      description: Name of the model
+      type: string
+  results:
+    - name: KFP_RUN_ID
+      description: Run ID of the Data Science Pipeline
+  steps:
+  - name: execute-doc-ingestion
+    workingDir: $(workspaces.output.path)/$(params.WORK_DIRECTORY)
+    image: quay.io/rlundber/mlops-ct-cicd:0.4
```

Comment:
The image will have to go in a BU repo. Not absolutely necessary for development, but let's not forget it.

Reply:
ah it is something we use in ML500, just changed to UBI9, it worked well with that too :)